### DISCUSSION TITLE
How to listen to `selected` event at component level?

### DISCUSSION THREAD

**Discussion by ngoc199** [GrapesJS/grapesjs#5054] [Q&A] [answered]
An event `component:selected` is fired whenever a component is selected. However, it is at the editor level.

I want to listen to my custom component's `selected` event at the component level.

I don't want to check the component type for every selected component, but my custom component only.

Does GrapesJS have any events that could achieve this?

**Comment by artf** [comment, marked as answer]
When the component is selected there is a local `status` change that you can subscribe and check for the `selected` value
```js
init() {
 this.on('change:status', (m, status) => {
   if (status === 'selected') console.log('component selected')
 })
}
```
